.github/changed-files.yml

```diff
@@ -12,6 +12,23 @@ linux:
   - 'BUILD.bazel'
   - '.bazelrc'
   - '.bazelversion'
+windows:
```

Comment:
By the way, the only reason I use this complicated setup is so that I can mark the Android / Linux / iOS workflows as required. You can also use https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

This means you only need one job.

Reply:
I found this a little bit confusing because I'm not used to it. Are you saying I don't need to add a `windows` entry here? Because the jobs in `windows-ci.yml` are meant to test every possible build flag combination, so this setup is useful to me at least. Please let me know what parts are unnecessary, because I'm learning about it yet.